packages/ui-app/src/Input.tsx

```diff
@@ -2,41 +2,44 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { BareProps } from './types';
+import { BareProps, SUIEvent } from './types';
 
 import React from 'react';
 import SUIInput from 'semantic-ui-react/dist/commonjs/elements/Input/Input';
 
 import isUndefined from '@polkadot/util/is/undefined';
 
 import Labelled from './Labelled';
+import Notification from './Notification';
 
 type Input$Type = 'number' | 'password' | 'text';
 
 type Props = BareProps & {
   children?: React.ReactNode,
   defaultValue?: any,
+  error?: React.ReactNode,
   icon?: any, // node?
+  info?: React.ReactNode,
   isAction?: boolean,
   isDisabled?: boolean,
   isEditable?: boolean,
   isError?: boolean,
   isHidden?: boolean,
   label?: any, // node?
   max?: any,
+  maxLength?: number,
   min?: any,
   name?: string,
   onChange: (value: string) => void,
+  onKeyDown?: (event: SUIEvent) => void,
```

Comment:
Don't pass the SUIEvent up to components using this - rather deconstruct the value in here (i.e. this component knows it uses SUI), calling components should not.